src/panels/lovelace/cards/hui-area-card.ts

```diff
@@ -236,21 +236,6 @@ export class HuiAreaCard extends LitElement implements LovelaceCard {
     }
   );
 
-  private _getCameraEntity = memoizeOne(
-    (
-      entities: HomeAssistant["entities"],
-      areaId: string
-    ): string | undefined => {
-      const cameraFilter = generateEntityFilter(this.hass, {
-        area: areaId,
-        entity_category: "none",
-        domain: "camera",
-      });
-      const cameraEntities = Object.keys(entities).filter(cameraFilter);
-      return cameraEntities.length > 0 ? cameraEntities[0] : undefined;
-    }
-  );
```

Comment:
Removing a default camera will introduce a breaking change to current area card setups